### PR TITLE
DO NOT MERGE - Hearings Welsh and Yes No - Should Go Live after Hearings PR - 3201 and Welsh Yes No PR

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/EUI-8956

### Change description ###
Remove the Welshify changes from the Hearings PR

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by udaydenduluri33 — APPROVED** [review]

File: src/hearings/containers/request-hearing/hearing-judge/hearing-judge.component.html [hunk 1]
```diff
@@ -43,7 +43,7 @@ <h1 class="govuk-fieldset__heading">{{ 'Name of the judge' | rpxTranslate }}</h1
               <input class="govuk-radios__input" id="noSpecificJudge" name="specificJudge"
                 (change)="showSpecificJudge('No');" formControlName="specificJudge" type="radio" value="No"
                 data-aria-controls="conditional-judge-type">
-              <label class="govuk-label govuk-radios__label" for="noSpecificJudge">{{ 'No' | rpxTranslate }}</label>
+              <label class="govuk-label govuk-radios__label" for="noSpecificJudge">{{ 'Do you want a specific judge?' | rpxTranslate : null : 'No' }}</label>
```
Comment (connorpgpmcelroy) [inline src/hearings/containers/request-hearing/hearing-judge/hearing-judge.component.html:46]: Minor comment: 'Do you want a specific judge?' especially as it is repeated should be a string constant/variable

**Review by connorpgpmcelroy — APPROVED** [review]
Approved with minor comment

File: src/hearings/containers/request-hearing/hearing-requirements/hearing-requirements.component.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import { AfterViewInit, Component, HostListener, OnDestroy, OnInit } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { Store } from '@ngrx/store';
+import { ValidatorsUtils } from 'src/hearings/utils/validators.utils';
```
Comment (connorpgpmcelroy) [inline src/hearings/containers/request-hearing/hearing-requirements/hearing-requirements.component.ts:4]: src used here

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Minor change requested

File: src/hearings/containers/request-hearing/hearing-requirements/hearing-requirements.component.ts (hunk 2 above)
Comment (DavidJayakumar) [reply src/hearings/containers/request-hearing/hearing-requirements/hearing-requirements.component.ts]: done

**Review by udaydenduluri33 — APPROVED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by connorpgpmcelroy — DISMISSED** [review]
LGTM

**Review by udaydenduluri33 — APPROVED** [review]

**Comment by DavidJayakumar** [comment]
3429 is the latest. so closing this PR

**Pull request closed without merging**